docs/studios/unity/tutorials/github_actions.md

```diff
@@ -0,0 +1,309 @@
+            Library-
+
+      # Test
+      # See https://game.ci/docs/github/test-runner for more information
+      - name: Run tests
+        uses: game-ci/unity-test-runner@v2
+        env:
+          UNITY_LICENSE: ${{ secrets.UNITY_LICENSE }}
+        with:
+          customParameters: -RG_BOT ${{ secrets.RG_BOT }} -RG_API_KEY ${{ secrets.RG_API_KEY }} -RG_HOST ${{ secrets.RG_HOST }}
+          githubToken: ${{ secrets.GITHUB_TOKEN }}
+
+      # Output
+      - uses: actions/upload-artifact@v3
+        with:
+          name: Build
+          path: build
+```
+
+:::info
+
+Note that this will actually run all of your tests, not just the QA Bot test. To configure this workflow to only
+run the QA test, see the [Unity Test Runner Command Line docs](https://docs.unity3d.com/Packages/com.unity.test-framework@2.0/manual/reference-command-line.html),
+which has filter options that can be passed within `customParameters` within the `Run tests` step above.
+
+:::
+
+## Configure GitHub Actions
+
+The final piece on the GitHub actions side is to configure the secrets that are used in the workflow file above. Visit the secrets
+page for your project by going to your **Repo** > **Settings** > **Secrets and variables** > **Actions**. Add the following secrets
+by clicking the green **New repository secret** button.
+
+1. `RG_HOST` - `https://play.regression.gg`
+2. `RG_API_KEY` - Your API key from Regression Games, which can be found in your [account settings](https://play.regression.gg/account).
+                  Full instructions for this can be found in the [Authentication docs.](studios/api_reference/authentication).
+3. `RG_BOT` - The ID of the bot you want to run. Within the [Bot Manager page](https://play.regression.gg/bots), you can see the ID
+              for a bot in its details below the name of the bot.
+
+![Bot ID](./github_actions_images/bot_id.png)
+
+Once you are done adding these secrets, your secrets section should look similar to the following:
+
+![Secrets for GitHub Actions](./github_actions_images/secrets.png)
+
+## Add a Test to Your Unity Project
+
+The QA Bots are started via a the [Unity Test Runner as a Play Mode test](https://docs.unity3d.com/2019.1/Documentation/Manual/PlaymodeTestFramework.html). 
+In this testing mode, the game starts as if you are playing it, and then terminates once the function finishes.
+
+Complete the following steps to setup the required test - feel free to skip some of these steps if you have existing Play tests
+in your project.
+
+#### 1. Add your scenes to your build
+
+Before adding the test, make sure your desired test scenes are placed inside your build. You can add your scenes to the build
+using the menu option **File** > **Build Settings**.
+
+![Build settings](./github_actions_images/build.png)
+
+#### 2. Configure Test Runner for your project
+
+Next, we need to configure the Unity Test Runner for your project. To do this, go to **Window** > **General** > **Test Runner**.
```

Comment:
I feel like we should have our own RG specific extension menu option here that does this step for them. (The creating the asmdef file parts)

Reply:
does this task exist ? If so I'll resolve this comment

Reply:
Added here: https://linear.app/regression-games/issue/REG-1240/create-menu-option-in-unity-to-automatically-setup-test-files